### PR TITLE
Abstraction levels

### PR DESCRIPTION
**Pull request by nerdinand**
New slides on the topic of Levels of Abstraction. 
The idea is to introduce the first level of abstraction which we encounter in the course, which is variables.
It also gives an outlook on the upcoming levels, Methods and Classes.
This would slot in as the last slideset in the 3rd lesson, after Arrays.

### CODE REVIEW

File: slides/abstraction-levels/index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            </div>
+            <div class="flex-column-200px">
+              <img class="no-border" src="../images/logo-cropped-small.png">
+            </div>
+          </div>
+          <aside class="notes">
+          </aside>
+        </section>
+
+        <section>
```
Comment (lxxxvi) [inline slides/abstraction-levels/index.html:46]: To me, this slide _looks crooked_ due to multiple "text-center-origins" (not sure if this is a real thing). May be we can clean that up by not using "left-quote, right-picture" style. What about this approach?

Screenshot 2021-01-05 at 10 42 33

Code

```html
        <section>
          <h2>Abstraction</h2>
          <img src="images/KnuthAtOpenContentAlliance.jpg" height="150px">
          <blockquote>
            "The psychological profiling [of a programmer] is mostly the ability to shift levels of abstraction, from low level to high level. To see something in the small and to see something in the large."<br />
            <em class="text-xl text-gray-500">Donald Knuth - Computer Scientist Extraordinaire</em>
          </blockquote>
          <div class="text-lg">
            (Source: <a href="https://en.wikiquote.org/wiki/Donald_Knuth">Wikiquote</a>)
          </div>
        </section>
```

**Review by lxxxvi — COMMENTED** [review]
@nerdinand 
Content-wise I think it's a masterpiece. 🥇 
Layout-wise I have one proposal, see it in the comments.

File: slides/abstraction-levels/index.html (hunk 1 above)
Comment (nerdinand) [reply slides/abstraction-levels/index.html:46]: The only downside is that the picture is tiny this way ;)

Comment (nerdinand) [reply slides/abstraction-levels/index.html:46]: hmm i guess i'll fiddle with it a bit

**Review by lxxxvi — APPROVED** [review]
Magnificent work. 🚀

**Comment by hharen** [comment]
Nice layout ;) This is just a detail but wouldn't we add the border colour from black to red? It looks like a death notice with the black frame.

**Comment by nerdinand** [comment]
@hharen haha, you're right, i'll change that

**Pull request merged**